docs/ide-usage.md

```diff
@@ -68,8 +68,10 @@ The following plugins are needed:
      * HTML: `Web → HTML Files → Editor → Indent using spaces`.
      * CSS: `Web → CSS Files → Editor → Indent using spaces`.
      * XML: `XML → XML Files → Editor → Indent using spaces`.
-   * Validation: We do not validate HTML, JSP, and XML.<br>
-     `Validation` → uncheck the `Build` option for `HTML Syntax Validator`, `JSP Content Validator`, `JSP Syntax Validator`, and `XML Validator`.
+   * Validation: 
+     * We do not validate HTML, JSP, and XML.`Validation` → uncheck the `Build` option for `HTML Syntax Validator`, `JSP Content Validator`, `JSP Syntax Validator`, and `XML Validator`.
+     * Disable JavaScript validation for node_modules folder. `Validation` → click the `...` settings button for `JavaScript Validation` → if `Exclude Group` is not already in the list then click `Add Exclude Group...` → `Exclude Group` → `Add Rule...` → select `Folder or file name` → `Next` → `Browse Folder...` → navigate to the `node_modules` folder → `OK` → then click on `Finish`.
```

Comment:
Hmm. That's ok lets just keep it as `Validation` for now. It will corrected from #7136. So no worries but it might confuse new contributors who goes through these steps. About the bug will wait for some core team members and ask their opinion. @whipermr5 as @Yogitha6  said there is a [bug here](https://bugs.eclipse.org/bugs/show_bug.cgi?id=455714) and i think( based on date) its reopened.